### PR TITLE
ci(Actions): implements the scripts to use actions

### PR DESCRIPTION
**Pull request by rapahaeru**
## Description
Put the description of the task here.

## Setup
to view the components behavior, run `yarn storybook`

## Review guide
- [ ] Coverage (coverage status should not regress)\
      - `yarn test:coverage`
- [ ] Unit tests (yarn test:components)
- [ ] Regression \
      - first start the storybook for regression tests(and keep it open): ` yarn test:regression:storybook`; \
      - then run the regression tests: `yarn test:regression`
- [ ] Code review

### CODE REVIEW

**Review by rodrigoclemente — APPROVED** [review]

File: .github/workflows/actions-development.yml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```
Comment (tcelestino) [inline .github/workflows/actions-development.yml:2]: **suggestion:** remove it! :sweat_smile:

File: .github/workflows/actions-publish.yml [hunk 2]
```diff
@@ -0,0 +1,63 @@
+# This workflow will do a clean install of node dependencies, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
```
Comment (tcelestino) [inline .github/workflows/actions-publish.yml:2]: **suggestion:* remove it :smile:

File: .github/workflows/actions-publish.yml [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+    strategy:
+      matrix:
+        node-version: [12.16.1]
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
```
Comment (tcelestino) [inline .github/workflows/actions-publish.yml:48]: it similiar my comments about the _matrix_ use.

File: .github/workflows/actions-publish.yml [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
```
Comment (tcelestino) [inline .github/workflows/actions-publish.yml:24]: it similiar my comments above

**Review by tcelestino — CHANGES_REQUESTED** [review]

File: .github/workflows/actions-development.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    strategy:
+      matrix:
+        node-version: [12.16.1]
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
```
[the remark this answers is not in the document]
Comment (rapahaeru) [reply .github/workflows/actions-development.yml:22]: Quantum need to be updated in some packages. One of them crashes if node version is greater the 12.16.1.

**Review by tcelestino — APPROVED** [review]

**Review by rodrigoclemente — APPROVED** [review]

**Review by pedrosouza — APPROVED** [review]

**Pull request merged**